### ISSUE TITLE
Close pop-up after opening the link

### ISSUE THREAD

**Issue by klorydryk** [te-online/files_linkeditor#64]
After opening a link, when coming back to the Nextcloud tab, the link editor pop-up is still open. It should be closing after opening the tab, as his job is done there.

**Comment by te-online** [comment]
Hey @klorydryk Thanks for your feedback. On first glance, this is on purpose; “a feature” so to speak. I agree, however, that the “confirmation” modal could close automatically – maybe you're only talking about that one 🤔

**Comment by klorydryk** [comment]
Yes, I'm talking about this “confirmation” modal that could close once the user clic on the "visit the link".
